app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/spark/Ingest.scala

```diff
@@ -323,6 +326,13 @@ object Ingest extends SparkJob with LazyLogging with Config {
     implicit def asS3Payload(status: IngestStatus): String = S3IngestStatus(sceneId, status).asJson.noSpaces
 
     try {
+      Integer.valueOf(Properties.envOrElse("AWS_BATCH_JOB_ATTEMPT", "0")) match {
+        case 3 => {
```

Comment:
This compilation failure is already in CI, but I was wondering if we could just match on strings here and avoid casting to integer altogether?

```
[error] /opt/raster-foundry/app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/spark/Ingest.scala:330:14: type mismatch;
[error]  found   : Int(3)
[error]  required: Integer
[error]         case 3 => {
[error]              ^
[error] one error found
[error] (Compile / compileIncremental) Compilation failed
[error] Total time: 22 s, completed Apr 27, 2018 2:49:12 PM
```

Reply:
I made that switch after compiling and testing locally... I'll go back to matching on strings, which was fine.